packages/chat-ui/src/components/CreateProfileModal.tsx

```diff
@@ -190,19 +190,12 @@ export function CreateProfileModal(props: Partial<ModalProps>) {
   }
 
   useEffect(() => {
-    if (loadingDelegate || (props.isOpen && !loadingNeeds && needsTopOff)) {
+    if (loadingDelegate || (props.isOpen && !loadingNeeds && needsInit)) {
```

Comment:
Still requiring a hot wallet since when they upload their pfp we need to approve multiple tx for shdw to work. bad ux, might as well have the hot wallet.